lib/src/extensions/rect.dart

```diff
@@ -1,7 +1,8 @@
 import 'dart:math';
-import 'dart:ui';
+import 'dart:ui' hide Offset;
```

Comment:
this should not conflict with importing OffsetExtensions from a file called 'offset.dar'? I am confused